bullet_train-themes/app/helpers/current_theme_helper.rb

```diff
@@ -1,4 +1,4 @@
-module ThemeHelper
+module CurrentThemeHelper
```

Comment:
Not a big deal, but it seems kind of odd to me that we're renaming things to accommodate a "competing" project. Is integration with JumpstartPro one of the explicit goals of this effort?

Reply:
Yep, open to suggestions though

Reply:
I guess I don't have much of a suggestion other than "let them sort it out" which probably isn't really helpful if the goal is for _us_ to sort it out.

Reply:
I think the broader question is whether ThemeHelper is that good a name in the first place for this.

I can also keep it at ThemeHelper here and maybe add the step in the jumpstart pro instructions to rename their own ThemeHelper to something else?

Reply:
Yeah, that's a good question. 🤔